authentication/index.adoc

```diff
@@ -0,0 +1,65 @@
+[id="authentication-authorization-overview"]
+= Overview of authentication and authorization
+include::modules/common-attributes.adoc[]
+:context: authentication-overview
+
+toc::[]
+
+
+[id="authentication-overview"]
+== Overview of authentication in {product-title}
+To keep an {product-title} cluster secure, a cluster administrator can configure user authentication and ensure only approved users access the cluster.
+
+To interact with an {product-title} cluster, users must authenticate to it by providing one of the following information in their requests to the {product-title} API:
+
+* An OAuth access token - Users can get an OAuth token from:
++
+ ** An `Authorization: Bearer…​` header.
+ ** A WebSocket subprotocol header for the web socket requests.
+ ** The built-in OAuth server through specific endpoints.
++
+Users can also xref:../authentication/managing-oauth-access-tokens.adoc#managing-oauth-access-tokens[view and manage their own OAuth acces tokens].
+
+* An X.509 client certificate - You require an HTTPS connection to the API server to use the X.509 client certificates for authentication.
+
+[NOTE]
+====
+If you do not present a valid access token or certificate, your request is unauthenticated and you receive an HTTP 401 error.
+====
+An administrator can configure authentication through the following tasks:
+
+* Configure an identity provider: You can define any xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[supported identity provider in {product-title}] and add it to your cluster. To specify an identity provider, you must create a custom resource (CR) that describes the identity provider.
+* xref:../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[Configure the internal OAuth server]: The {product-title} master includes a built-in OAuth server that determines the user’s identity from the configured identity provider and creates access tokens. You can configure the token duration and inactivity timeout, and customize the internal OAuth server URL.
+* xref:../authentication/configuring-oauth-clients.adoc#oauth-register-additional-client_configuring-oauth-clients[Register an OAuth client]: {product-title} includes many xref:../authentication/configuring-oauth-clients.adoc#oauth-default-clients_configuring-oauth-clients[default OAuth clients]. You can register and configure additional OAuth clients.
```

Comment:
I'd suggest changing "many" to "several", since there are really only 2 OAuth clients, which I don't think is necessarily many.